test/lib/y2users/user_validator_test.rb

```diff
@@ -75,6 +75,15 @@
         end
       end
 
+      context "and using invalid characters" do
+        # See UserValidator::CHAR_REGEXP
+        let(:username) { "abc++" }
+
+        it "includes a fatal issue for the username length" do
```

Comment:
Description looks wrong. What about using something like 


```suggestion
        it "includes a fatal issue for using not allowed characters" do
```

Reply:
copy and paste error, thanks for catch